src/MissionEditor/SimpleItemEditor.qml

```diff
@@ -119,6 +119,13 @@ Rectangle {
                 onClicked:  editorRoot.moveHomeToMapCenter()
                 anchors.horizontalCenter: parent.horizontalCenter
             }
+
+            QGCButton {
```

Comment:
There already is a center map tool on the Plan view. This show go there not in the item editors. The name should be "Current item".